osu.Game/Rulesets/Judgements/Judgement.cs

```diff
@@ -12,30 +13,68 @@ namespace osu.Game.Rulesets.Judgements
     public class Judgement
     {
         /// <summary>
-        /// The default health increase for a maximum judgement, as a proportion of total health.
-        /// By default, each maximum judgement restores 5% of total health.
+        /// The score awarded for a small bonus.
         /// </summary>
-        protected const double DEFAULT_MAX_HEALTH_INCREASE = 0.05;
+        public const int SMALL_BONUS_SCORE = 10;
 
         /// <summary>
-        /// The maximum <see cref="HitResult"/> that can be achieved.
+        /// The score awarded for a large bonus.
         /// </summary>
-        public virtual HitResult MaxResult => HitResult.Perfect;
+        public const int LARGE_BONUS_SCORE = 50;
+
+        /// <summary>
+        /// The default health increase for a maximum judgement, as a proportion of total health.
+        /// By default, each maximum judgement restores 5% of total health.
+        /// </summary>
+        protected const double DEFAULT_MAX_HEALTH_INCREASE = 0.05;
 
         /// <summary>
         /// Whether this <see cref="Judgement"/> should affect the current combo.
         /// </summary>
+        [Obsolete("Has no effect. Use HitResult members instead (e.g. use small-tick or bonus to not affect combo).")] // Can be removed 20210328
         public virtual bool AffectsCombo => true;
 
         /// <summary>
         /// Whether this <see cref="Judgement"/> should be counted as base (combo) or bonus score.
         /// </summary>
+        [Obsolete("Has no effect. Use HitResult members instead (e.g. use small-tick or bonus to not affect combo).")] // Can be removed 20210328
         public virtual bool IsBonus => !AffectsCombo;
 
+        /// <summary>
+        /// The maximum <see cref="HitResult"/> that can be achieved.
+        /// </summary>
+        public virtual HitResult MaxResult => HitResult.Perfect;
+
+        /// <summary>
+        /// The minimum <see cref="HitResult"/> that can be achieved - the inverse of <see cref="MaxResult"/>.
+        /// </summary>
+        public HitResult MinResult
+        {
+            get
+            {
+                switch (MaxResult)
+                {
+                    case HitResult.SmallBonus:
```

Comment:
might be good to document these along with the types table (for the corresponding "max" results) as they are hard coupled with the max result.

Reply:
I've added a wiki page: https://github.com/ppy/osu/wiki/Scoring . Please check and point out any issues you have with it.